table_helper.hh

```diff
@@ -24,11 +24,15 @@
 
 #include <seastar/util/gcc6-concepts.hh>
 #include <seastar/core/apply.hh>
-#include "cql3/statements/modification_statement.hh"
```

Comment:
Please make sure it does not break audit in enterprise. If I remember correctly table_helper was used there.

Reply:
You're right, the "enterprise" branch no longer builds with this change. I get a bizaare compilation error from do_with(). @avikivity can you please look into this and if needed, submit a new version of this series? Thanks.

Reply:
When I merge master into enterprise I will fix things up in the merge commit. In general contributors do not need to consider invisible branches.